libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -21,6 +21,28 @@ namespace Microsoft.Bot.Builder.Teams
     /// </summary>
     public static class TeamsInfo
     {
+        /// <summary>
+        /// Gets the details for the given meeting participant. This only works in teams meeting scoped conversations. 
+        /// </summary>
+        /// <param name="turnContext">Turn context.</param>
+        /// <param name="meetingId">The id of the Teams meeting. TeamsChannelData.Meeting.Id will be used if none provided.</param>
+        /// <param name="participantId">The id of the Teams meeting participant. From.AadObjectId will be used if none provided.</param>
+        /// <param name="tenantId">The id of the Teams meeting Tenant. TeamsChannelData.Tenant.Id will be used if none provided.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        /// <remarks>InvalidOperationException will be thrown if meetingId, participantId or tenantId have not been
+        /// provided, and also cannot be retrieved from turnContext.Activity.</remarks>
+        /// <returns>Team participant channel account.</returns>
+        public static async Task<TeamsParticipantChannelAccount> GetParticipantAsync(ITurnContext turnContext, string meetingId = null, string participantId = null, string tenantId = null, CancellationToken cancellationToken = default)
```

Comment:
Since this is only valid in a meeting context, this might be better named `GetMeetingParticipant`